__tests__/app.test.js

```diff
@@ -52,4 +52,37 @@ describe('APP', () => {
             })
         });
     });
+    describe('GET /api/articles/:article_id', () => {
+        test('should repsond with a status 200 and an object with the appropriate keys', () => {
+            return request(app)
+            .get('/api/articles/3')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.article).toHaveProperty('author', expect.any(String))
+                expect(response.body.article).toHaveProperty('title', expect.any(String))
+                expect(response.body.article).toHaveProperty('article_id', expect.any(Number))
+                expect(response.body.article).toHaveProperty('body', expect.any(String))
+                expect(response.body.article).toHaveProperty('topic', expect.any(String))
+                expect(response.body.article).toHaveProperty('created_at', expect.any(String))
+                expect(response.body.article).toHaveProperty('votes', expect.any(Number))
+                expect(response.body.article).toHaveProperty('article_img_url', expect.any(String))
+        })
```

Comment:
this works fine.
Going forward I would have a look into toMatchObject or objectContaining jest matchers they will allow for more robust testing,